ibis/backends/postgres/tests/test_client.py

```diff
@@ -30,6 +30,7 @@
     'IBIS_TEST_POSTGRES_DATABASE', 'ibis_testing'
 )
 IBIS_POSTGRES_HOST = os.environ.get('IBIS_TEST_POSTGRES_HOST', 'localhost')
+IBIS_POSTGRES_PORT = os.environ.get('IBIS_TEST_POSTGRES_PORT', '5432')
```

Comment:
Shouldn't this be `5433`?

Reply:
Hm, yeah. Good catch.

Reply:
Actually, I think this should default to the default postgres port, since it's perfectly fine to test this against a standard postgres install.

Reply:
Right.